.github/pull_request_template.md

```diff
@@ -1,34 +1,24 @@
-# Description
+## Description
 
-Please include a summary of the feature or bug and the changes/implementations to tackle it. Please also include relevant motivation and context. List any dependencies that are required for this change.
+Include a list of changes made or features created. Include any relevant screenshots or links.
 
-## Type of change
+## Type of Change
 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Breaking change (change that introduces a bug or causes app to crash, but that we plan on fixing later)
 - [ ] This change requires a documentation update
 
-# How Has This Been Tested?
+## How Has This Been Tested?
 
-Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+Provide any test suites/filenames that you had to create/change.
 
 - [ ] Test A
 - [ ] Test B
 
-**Test Configuration**:
-* Firmware version:
-* Hardware:
-* Toolchain:
-* SDK:
 
-# Checklist:
+## Checklist
 
-- [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
-- [ ] My changes generate no new warnings
-- [ ] I have added tests that prove my fix is effective or that my feature works
-- [ ] New and existing unit tests pass locally with my changes
```

Comment:
Shouldn't we keep these checkboxes to add in unit tests, now that we are developing them?

Reply:
It's probably redundant, since the testing section will cover that?